Makefile.kokkos

```diff
@@ -1174,6 +1179,11 @@ ifeq ($(KOKKOS_INTERNAL_USE_HIP), 1)
   endif
 endif
 
+ifeq ($(KOKKOS_INTERNAL_ENABLE_DESUL_ATOMICS), 1)
+  tmp := $(call kokkos_append_header,"$H""define KOKKOS_ENABLE_IMPL_DESUL_ATOMICS")
+endif
+
+tmp := $(call kokkos_append_header,"$H""define KOKKOS_ENABLE_LIBDL")
```

Comment:
Not sure why this is shown as new here. The LIBDL thing is in master right now, and yes the Makefile.kokkos does this unconditionally since we only support platforms with Makefiles which have it.

Reply:
It's already in the file, see https://github.com/kokkos/kokkos/blob/9fee92f46596334850edcf4a099f1a76eb084c53/Makefile.kokkos#L589. So this is a duplication.